### DISCUSSION TITLE
PDNS recursor and PDNS authroitive

### DISCUSSION THREAD

**Discussion by ahmedhkhalil** [PowerDNS/pdns#14803] [General]
Dears,
i have two power dns auth Server where i am hosting my domain , i need to enable caching on them i found that power dns recursorbest match but i am facing below issues : 
1) recursoris responding to any request even domains that not exist in forward zone 
2) when i validate dnssec on recursor i got error message that it's not secure as recursor server  is not authorities to handle this domain

**Comment by omoerbeek** [comment]
You cannot use a *recursor* as a front end to an authoritative server. Maybe dnsdist is what you are looking for? By the way, the authoritative server does cache itself (for the zones it is authoritative for).

**Comment by ahmedhkhalil** [comment]
@omoerbeek  so how can i increase bandwidth and number of requests , should i create several auth servers for that and dnsdist to load balancer between them , or can i use auth binaries and use it as caching only mode if yes how can i do that 
and what is main functionality of recursor
